### DISCUSSION TITLE
GitHub Copilot plugin not available on android studio

### DISCUSSION THREAD

**Discussion by Oluwafemijohn1** [community/community#7400] [Copilot Conversations] [answered]
I was able to install the copilot plugin on IntelliJ but could not find it on android studio meanwhile, I just updated the android studio. Does it mean it's not available on android studio yet?

**Comment by FerUribeM** [comment, marked as answer]
Currently only for Android Studio Canary

**Reply by balakstats** [reply]
Which, unfortunately, doesn't seem to support Kotlin.

**Reply by Vahalaru** [reply]
It says it doesn't support kotlin. I can verify that if you upgrade to canary, copilot will work using kotlin. I haven't tried with other languages yet but it seemed to work fairly well,  albeit a little bit or miss. That said after tinkering around to get it to work with android studio I have only been able to work on my program for about 20 minutes.

**Reply by wiesys** [reply]
I can verify that too and from my point of view, it works very well :)

**Reply by yash10019coder** [reply]
How much time it will take to come support for stable android studio release of github copilot?

**Reply by manuelduarte077** [reply]
If it works I use kotlin, in intellij idea and everything fine, I also use it for java

**Reply by ChrisClaude** [reply]
I'm using dart on Android Studio Artic Fox. I can't wait to have GitHub Copilot available.

**Reply by manuelduarte077** [reply]
To use copilot in android you have to use the preview version

**Reply by helderpgoncalves** [reply]
Actually, there is answer for this. You can use Android Studio for development but you can open tab with vscode using copilot on the same project to use copilot :)

**Reply by manuelduarte077** [reply]
haha good move 🔥

**Reply by harismuntazir** [reply]
> Actually, there is answer for this. You can use Android Studio for development but you can open tab with vscode using copilot on the same project to use copilot :)

I am actually doing the same procedure as of now, and to be honest it works quite well. VS Code + Android Studio

**Reply by soumwadeep** [reply]
When It Will Be Available For Android Studio Windows

**Comment by ilyassesalama** [comment]
The accepted answer is no longer valid, GitHub Copilot is available on Android Studio now.

![image](https://user-images.githubusercontent.com/46769766/169705351-6c13daeb-9432-49a9-9012-7eca61ce9189.png)

**Reply by manuelduarte077** [reply]
in fact recently, it is already available

**Reply by AhmedDoma2020** [reply]
I can't fount it ,
which version u download ?

**Reply by manuelduarte077** [reply]
https://plugins.jetbrains.com/plugin/17718-github-copilot

**Reply by ber4444** [reply]
too bad kotlin is not a supported language
